diff -uraN Pillow-8.4.0/setup.py omv-Pillow-8.4.0/setup.py
--- Pillow-8.4.0/setup.py	2021-10-15 08:27:22.000000000 +0200
+++ omv-Pillow-8.4.0/setup.py	2021-11-03 00:48:52.194178315 +0100
@@ -513,10 +513,10 @@
 
             ft_prefix = None
 
-            if prefix:
+#            if prefix:
                 # add Homebrew's include and lib directories
-                _add_directory(library_dirs, os.path.join(prefix, "lib"))
-                _add_directory(include_dirs, os.path.join(prefix, "include"))
+#                _add_directory(library_dirs, os.path.join(prefix, "lib"))
+#                _add_directory(include_dirs, os.path.join(prefix, "include"))
                 _add_directory(
                     include_dirs, os.path.join(prefix, "opt", "zlib", "include")
                 )
